src/components/Cell.vue

```diff
@@ -241,6 +253,9 @@ export default {
       }
     }
   }
+  select {
```

Comment:
Please add a class to the select component (maybe `.grid-select`) and set style using this class.